AhoCorasick.py

```diff
@@ -0,0 +1,133 @@
+# implementing AhoCorasick using adjacency list
+import collections

```

Comment:
please include this file in the relevant folder 

rename the file name as Aho_Corasick.py